### ISSUE TITLE
Plugin prevents typing a pipe on french Mac keyboards

### ISSUE THREAD

**Issue by FGM** [syllant/idea-plugin-linessorter#6]
Just notices this on PhpStorm 2016.2 : the plugin uses the default key combination used to input a pipe on a french Mac keyboard, causing very strange behaviors, like all the code getting reorganized when you are typing that very common character in a || in PHP, or a | in Bash.

Remapping from Preferences addresses the problem, but maybe it could avoid that combination.

**Comment by syllant** [comment]
You're right, finding available shortcuts is now a hell for IntelliJ products and plugin developers should refrain from assigning default shortcuts :) Should a next version be published, this fix will be definitively included.

Thanks
